docs/thinlto.md

```diff
@@ -0,0 +1,42 @@
+# Extracting Corpora for ThinLTO-enabled Projects
+
+These steps are tailored for the inlining problem.
```

Comment:
They would work otherwise, right? (with the right adjustments). Maybe just say that these steps use inlining -Oz  as example. 

...but are they inlining-specific?